packages/engine-ui/src/components/CreateVehicle.js

```diff
@@ -14,9 +14,9 @@ const initialState = {
   volume: '',
   weight: '',
   timewindow: { start: null, end: null },
-  startPosition: { lat: 61.8172594, lon: 16.0561472, name: '' },
-  endPosition: { lat: undefined, lon: undefined, name: '' },
-  driver: { name: '', contact: '' },
+  startPosition: { lat: 61.8172594, lon: 16.0561472, name: undefined },
+  endPosition: { lat: undefined, lon: undefined, name: undefined },
+  driver: { name: undefined, contact: undefined },
```

Comment:
Input expects a string, as this will cause UI to throw controlled/uncontrolled error in console
![image](https://user-images.githubusercontent.com/21989632/96118478-d8a92800-0eeb-11eb-8271-34a8c7b6482c.png)